### PR TITLE
+ standard issue template

### LINKED ISSUE #3773: Support test project board
**Description**
_(Short description what is wrong, what area and where)_

Golem Version:
Golem-Messages version:
Electron version (if used):
OS:
Branch (if launched from source):
Mainnet/Testnet:

Description of the issue:

**Repro steps**
_(short steps description how it happens)_

**Actual result**
_(What is the result of this issue)_

**Expected result**
_(What is the expected result)_

**Logs**
_(Any additional information that could help to resolve the issue, which systems were checked to reproduce the issue)_

**Proposed Solution**
_(What could be a solution for that issue)_

### PR DESCRIPTION
**Pull request by shadeofblue**
closes https://github.com/golemfactory/golem/issues/3773

### CODE REVIEW

**Review by ederenn — COMMENTED** [review]
seems legit

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Please tell us if you encountered an issue with Golem
+title: ''
+labels: brass, bug
+assignees: ''
```
Comment (shadeofblue) [inline .github/ISSUE_TEMPLATE/bug-report.md]:
```suggestion
assignees: 'ZmijaWA'
```

**Comment by shadeofblue** [comment]
@ZmijaWA added you as the default assignee as per the "verifier" role in the "User support" flow doc

**Review by ZmijaWA — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+about: Please tell us if you encountered an issue with Golem
+title: ''
+labels: brass, bug
+assignees: 'ZmijaWA'
+
+---
+
+## Description
+
+**Golem Version**:
```
Comment (badb) [inline .github/ISSUE_TEMPLATE/bug-report.md:12]: Maybe add empty lines in this part?

**Review by badb — APPROVED** [review]

**Pull request merged**